background.ts

```diff
@@ -324,6 +294,77 @@ Electron.app.whenReady().then(async() => {
   }
 });
 
+/**
+ * Checks for the existence of the 'backend.lock' file and updates the
+ * 'backendIsLocked' variable accordingly. Emits the 'backend-locked-update'
+ * event to notify listeners about the current lock status.
+ *
+ * @returns A promise that resolves to a boolean that indicates if
```

Comment:
It'd be easier to read if the type was explicit, instead of being inside the prose description. (Of course, we'd still need a comment to say what it means…)
```jsdoc
@returns Whether the backend lock exists
```
```ts
async function doesBackendLockExists(): Promise<boolean>
```
(On second thought, given the function name I'm not sure if the `@returns` ended up being useful.)

Reply:
Agreed, it's not helpful here and the explicit type relays the same information more effectively.